src/ftsh_functions.c

```diff
@@ -1,69 +1,111 @@
 #include "ftsh.h"
```

Comment:
This file has tabs & spaces mixed. Please standardize on 4 spaces for indentation.

Reply:
fixed with astyle